### PR TITLE
Atlas edit weekly cycle proposal   governance poll   2024 09 16

### PR DESCRIPTION
**Pull request by blimpa**
Originating forum post: https://forum.makerdao.com/t/atlas-edit-weekly-cycle-proposal/25083

### CODE REVIEW

File: governance/polls/templates/Generic/Atlas Edit Weekly Cycle Proposal - September 16, 2024.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: Atlas Edit Weekly Cycle Proposal - September 16, 2024
+summary: Signal your support or opposition to this Atlas edit, which: 1) Updates the GSM Delay Exception for the Smart Burn Engine based on a request from BA Labs; 2) adds exemption for the Facilitator anonymity requirement for Ecosystem based on discussion with Governance Facilitators; 3) makes minor updates to language in some places to incorporate initial feedback on Atlas v2.
```
Comment (boet1) [inline governance/polls/templates/Generic/Atlas Edit Weekly Cycle Proposal - September 16, 2024.md:3]: The text in the summary causes a conflict due to the colons after the word **'which:'**. I suggest placing the entire text in quotation marks to resolve the issue without modifying the summary text.
```suggestion
summary: "Signal your support or opposition to this Atlas edit, which: 1) Updates the GSM Delay Exception for the Smart Burn Engine based on a request from BA Labs; 2) adds exemption for the Facilitator anonymity requirement for Ecosystem based on discussion with Governance Facilitators; 3) makes minor updates to language in some places to incorporate initial feedback on Atlas v2."
```

**Comment by boet1** [comment]
It seems the poll is in the wrong folder. It should be modified.

File: governance/polls/templates/Generic/Atlas Edit Weekly Cycle Proposal - September 16, 2024.md (hunk 1 above)
Comment (CivicSage3) [reply governance/polls/templates/Generic/Atlas Edit Weekly Cycle Proposal - September 16, 2024.md:3]: We can also just remove the conflicting signs, no?

File: governance/polls/Atlas Edit Weekly Cycle Proposal - September 16, 2024.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+title: Atlas Edit Weekly Cycle Proposal - September 16, 2024
+summary: Signal your support or opposition to this Atlas edit, which: 1) Updates the GSM Delay Exception for the Smart Burn Engine based on a request from BA Labs; 2) adds exemption for the Facilitator anonymity requirement for Ecosystem based on discussion with Governance Facilitators; 3) makes minor updates to language in some places to incorporate initial feedback on Atlas v2.
```
Comment (CivicSage3) [inline governance/polls/Atlas Edit Weekly Cycle Proposal - September 16, 2024.md:3]:
```suggestion
summary: Signal your support or opposition to this Atlas edit, which: 1) updates the GSM Delay Exception for the Smart Burn Engine based on a request from BA Labs; 2) adds an exemption for the Facilitator anonymity requirement for Ecosystem, based on discussion with Governance Facilitators; 3) makes minor updates to language in some places to incorporate initial feedback on Atlas v2.
```

File: governance/polls/Atlas Edit Weekly Cycle Proposal - September 16, 2024.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+## Review
+
+The community can vote in this poll to express support or opposition to the following Atlas Edit Weekly Cycle proposal:
+
+- [Atlas Edit Pull Request](https://github.com/makerdao/next-gen-atlas/pull/33)
+- [Proposal Discussion Thread](https://forum.makerdao.com/t/atlas-edit-weekly-cycle-proposal/25083)
+
+A brief summary of this Atlas Edit has been provided by the proposal author and is shown below:
+
+- *Updates GSM delay exception for Smart Burn Engine based on request from BA Labs. (This update has also been reviewed/approved by Ephy).*
```
Comment (CivicSage3) [inline governance/polls/Atlas Edit Weekly Cycle Proposal - September 16, 2024.md:37]:
```suggestion
- *Updates [GSM delay exception for Smart Burn Engine](https://sky-atlas.powerhouse.io/#A.1.8.2.2.8_Smart_Burn_Engine_Breaker_Exception-d82f2eb4-fe8b-4bc0-827e-9e3f859118b5|0db30758e055352c) based on a request from BA Labs. (This update has also been reviewed/approved by Ephy).*
```

File: governance/polls/Atlas Edit Weekly Cycle Proposal - September 16, 2024.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+- *Adds exemption for the Facilitator anonymity requirement for Ecosystem based on discussion with Governance Facilitators.*
```
Comment (CivicSage3) [inline governance/polls/Atlas Edit Weekly Cycle Proposal - September 16, 2024.md:38]:
```suggestion
- *Adds exemption for the Facilitator anonymity requirement for Ecosystem, based on discussion with Governance Facilitators.*
```

File: governance/polls/Atlas Edit Weekly Cycle Proposal - September 16, 2024.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+- This associated Pull Request will be merged into the Atlas.
+
+**If the votes for the 'No' option equal or exceed the votes for the 'Yes' option then no further action will be taken at this time.**
+
+---
+
+## Resources
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
```
Comment (CivicSage3) [inline governance/polls/Atlas Edit Weekly Cycle Proposal - September 16, 2024.md:53]:
```suggestion
If you are new to voting in the Sky Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
```

File: governance/polls/Atlas Edit Weekly Cycle Proposal - September 16, 2024.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+Additional information about the Governance process can be found in the [Maker Operational Manual](https://manual.makerdao.com).
+
+To add current and upcoming votes to your calendar, please see the [MakerDAO Governance Calendar](https://manual.makerdao.com/makerdao/calendars/governance-calendar).
```
Comment (CivicSage3) [inline governance/polls/Atlas Edit Weekly Cycle Proposal - September 16, 2024.md:57]:
```suggestion
To add current and upcoming votes to your calendar, please see the [Sky Governance Calendar](https://manual.makerdao.com/makerdao/calendars/governance-calendar).
```

File: governance/polls/Atlas Edit Weekly Cycle Proposal - September 16, 2024.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-09-16T16:00:00
+end_date: 2024-09-19T16:00:00
```
Comment (VoteWizard) [inline governance/polls/Atlas Edit Weekly Cycle Proposal - September 16, 2024.md:16]:
```suggestion
title: Atlas Edit Weekly Cycle Proposal - September 16, 2024
summary: Signal your support or opposition to this Atlas edit, which 1) Updates the GSM Delay Exception for the Smart Burn Engine based on a request from BA Labs, 2) adds exemption for the Facilitator anonymity requirement for Ecosystem based on discussion with Governance Facilitators, 3) makes minor updates to language in some places to incorporate initial feedback on Atlas v2.
discussion_link: https://forum.makerdao.com/t/atlas-edit-weekly-cycle-proposal/25083
parameters:
    input_format:
        type: single-choice
        abstain: [0]
    victory_conditions:
        - {
            type: 'and',
            conditions: [
                { type : plurality },
                { type : comparison, comparator : '>=', value: 10000 }
            ]
        }
        - {type : default, value : 2 }
    result_display: single-vote-breakdown
version: v2.0.0
options:
   0: Abstain
   1: Yes
   2: No
start_date: 2024-09-16T16:00:00
end_date: 2024-09-19T16:00:00
```

File: governance/polls/Atlas Edit Weekly Cycle Proposal - September 16, 2024.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+parameters:
+    input_format:
+        type: single-choice
+        abstain: [0]
+    victory_conditions:
+        - {
+            type: 'and',
+            conditions: [
+                { type : plurality },
+                { type : comparison, comparator : '>=', value: 10000 }
```
Comment (VoteWizard) [inline governance/polls/Atlas Edit Weekly Cycle Proposal - September 16, 2024.md:14]:
```suggestion
                { type : comparison, comparator : '>=', value: 20000 }
```

**Review by VoteWizard — APPROVED** [review]

**Pull request merged**